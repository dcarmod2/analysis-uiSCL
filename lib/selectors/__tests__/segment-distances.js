import selectSegmentDistances from '../segment-distances'

test('should get the distances correctly for the active modifications segments', () => {
  expect(
    selectSegmentDistances({
      project: {
        modifications: [
          {
            _id: '1',
            segments: [
              {
                geometry: {
                  type: 'LineString',
                  coordinates: [
                    [-77, 38],
                    [-77, 39]
                  ]
                }
              }
            ]
          }
        ]
      },
      queryString: {
        modificationId: '1'
      }
    }).map((v) => v.toFixed(8))
  ).toMatchSnapshot()
})
