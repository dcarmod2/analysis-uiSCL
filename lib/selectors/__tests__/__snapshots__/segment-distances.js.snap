// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should get the distances correctly for the active modifications segments 1`] = `
Array [
  "111.19508023",
]
`;
